test/unit/search/fm_index_cursor/fm_index_cursor_collection_test_template.hpp

```diff
@@ -404,7 +405,8 @@ TYPED_TEST_P(fm_index_cursor_collection_test, extend_const_char_pointer)
 
 TYPED_TEST_P(fm_index_cursor_collection_test, concept_check)
 {
-    EXPECT_TRUE(seqan3::fm_index_cursor_specialisation<TypeParam>);
+    EXPECT_TRUE((seqan3::detail::template_specialisation_of<TypeParam, seqan3::fm_index_cursor> ||
+    seqan3::detail::template_specialisation_of<TypeParam, seqan3::bi_fm_index_cursor>));
 }
```

Comment:
You can just remove the concept check :) checking if it is a specialisation of ... does not really do anything meaningful in this context.

The include of the fm_index_cursor.hpp may not be necessary also then.